### ISSUE TITLE
SS 4.2: Title + Displayed input group is broken

### ISSUE THREAD

**Issue by robbieaverill** [dnadesign/silverstripe-elemental#201]
Fresh install of SS 4.2.x-dev and elemental 2.x-dev:

![image](https://user-images.githubusercontent.com/5170590/37060462-40aa45a6-21f5-11e8-9f8d-25555e5a6d5c.png)

It should look like this (SS 4.0/4.1):

![image](https://user-images.githubusercontent.com/5170590/37060472-47b20ba4-21f5-11e8-9904-f963c0261abc.png)

---

SS 4.2 includes an update of Bootstrap 4 from beta to stable. It looks like the classes needed for the input group have changed in those versions.

E.g.:

```diff
- <span class="input-group-addon">
+ <span class="input-group-append">

- <label class="form-label-check">
+ <label class="input-group-text">
```

And add new CSS:

```scss
.input-group .input-group-append .input-group-text .form-check-input.checkbox {
  position: relative;
  margin-left: 0;
}
```

Result:
![image](https://user-images.githubusercontent.com/5170590/37060650-c7029b6c-21f5-11e8-9403-8fb29d2d949d.png)

We will need to implement this in a way that works on both versions of Bootstrap to support SS 4.0/4.1 and SS 4.2 and newer

**Comment by robbieaverill** [comment]
In #218 I've bumped the core version to SS 4.2 for the master branch (3.0.x-dev), so we can make that the SS 4.2 and newer branch

**Comment by robbieaverill** [comment]
Fix at #219
